sourcegraph/sourcegraph
# Code Insights: Remove legacy insight card (view package)

Based on https://github.com/sourcegraph/sourcegraph/pull/35925

This PR replaces the old API around insight card (the view package directory) with newly created and adopted in Code Insights pages `InsightCard` components. 

## Test plan
- Make sure that code insights card look good on the on-prem and cloud landing pages
- Make sure that templates work without any regression 



## App preview:

- [Web](https://sg-web-vk-remove-legacy-insight-card.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-kyyzwicbca.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
